zerver/tests/test_subs.py

```diff
@@ -3075,7 +3075,7 @@ def test_realm_admin_remove_others_from_unsubbed_private_stream(self) -> None:
         streams you aren't on.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=19,
+            query_count=21,
```

Comment:
These numbers increased because checking if you can send a message now requires a query, or something else?

Reply:
Ahh, this goes down just after, fine.